cmd/webhook/app/webhook.go

```diff
@@ -86,5 +85,23 @@ func NewServerWithOptions(log logr.Logger, opts options.WebhookOptions) (*server
 		MutationWebhook:   mutationHook,
 		ConversionWebhook: conversionHook,
 		Log:               log,
-	}, nil
+	}
+
+	return srv.Run(stopCh)
+}
+
+func NewServerCommand(stopCh <-chan struct{}) *cobra.Command {
+	opts := options.WebhookOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "cert-manager-webhook",
+		Short: fmt.Sprintf("Webhook for automated TLS controller for Kubernetes (%s) (%s)", util.AppVersion, util.AppGitCommit),
```

Comment:
This text doesn't seem right? `Webhook component providing API validation, mutation and conversion functionality for cert-manager` perhaps?